workflows/gen_crates.py

```diff
@@ -0,0 +1,66 @@
+import json
+import os
+import pathlib
+
+# pip install 'rocrate==0.4.0'
+from rocrate.rocrate import ROCrate
+
+THIS_DIR = pathlib.Path(__file__).absolute().parent
+
+ROOT = THIS_DIR / "sars-cov-2-variant-calling"
```

Comment:
I guess we want to avoid hardcoding this ?

Reply:
Done in 56191ba. Root directory is now a command line argument that can take 0 or more values, and defaults to the cwd.